pom.xml

```diff
@@ -12,7 +12,7 @@
 	<url>https://github.com/frankframework/frank-doc</url>
 
 	<properties>
-		<revision>3.3-SNAPSHOT</revision>
+		<revision>4.0</revision>
```

Comment:
For this repo, we have a pipeline that should automatically create this commit when releasing. 

You could, however, change this to `4.0-SNAPSHOW` with `HEAD` as tag.